subprojects/ide/src/main/java/org/gradle/plugins/ide/idea/model/Module.java

```diff
@@ -181,15 +221,20 @@ protected String getDefaultResourceName() {
     }
 
     protected Object configure(Path contentPath,
```

Comment:
Declare the binary compatibility breakage in the `subprojects/distributions/src/changes/accepted-public-api-changes.json` file. The content should be something like this (I copied it from the compatibility check report):
```
{
    "type": "org.gradle.plugins.ide.idea.model.Module",
    "member": "Method org.gradle.plugins.ide.idea.model.Module.configure(org.gradle.plugins.ide.idea.model.Path,java.util.Set,java.util.Set,java.util.Set,java.util.Set,java.util.Set,java.util.Set,java.lang.Boolean,org.gradle.plugins.ide.idea.model.Path,org.gradle.plugins.ide.idea.model.Path,java.util.Set,java.lang.String,java.lang.String)",
    "acceptation": "This method is not meant to be part of the public API"
},
{
    "type": "org.gradle.plugins.ide.idea.model.Module",
    "member": "Method org.gradle.plugins.ide.idea.model.Module.configure(org.gradle.plugins.ide.idea.model.Path,java.util.Set,java.util.Set,java.util.Set,java.util.Set,java.lang.Boolean,org.gradle.plugins.ide.idea.model.Path,org.gradle.plugins.ide.idea.model.Path,java.util.Set,java.lang.String,java.lang.String)",
    "acceptation": "This method is not meant to be part of the public API",
    "changes": [
        "Method has been removed"
    ]
}
```